src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -1,4 +1,4 @@
-{
+f{
```

Comment:
typo

Reply:
I HATE my Mac. The keyboard is BROKEN, like many other fellows.